registry/storage/garbagecollect.go

```diff
@@ -122,8 +137,18 @@ func MarkAndSweep(ctx context.Context, storageDriver driver.StorageDriver, regis
 			if err != nil {
 				return fmt.Errorf("failed to delete manifest %s: %v", obj.Digest, err)
 			}
+			for _, gcHandler := range opts.GCExtensionHandlers {
+				err := gcHandler.RemoveManifest(ctx, storageDriver, registry, obj.Digest, obj.Name)
```

Comment:
If I understand correctly, this is invoked after the manifest has already been deleted by the vacuum and is meant as a post processing step? Should it be renamed to reflect that, such as `OnManifestDeleted`?

Reply:
For ORAS GC Extension Handler, this method is responsible for going through all the artifacts indexed for sweeping. For each, it deletes the artifact manifest revision and the corresponding _refs folder. And finally it decrements the blob ref count map for the manifest and the blobs. I agree that the name is a bit misleading. I'll change it.